joinrun/src/test/scala/code/chymyst/test/StaticAnalysisSpec.scala

```diff
@@ -92,7 +95,14 @@ class StaticAnalysisSpec extends FlatSpec with Matchers with TimeLimitedTests {
         go { case a(Some(1)) + b(2) => }
       )
     }
-    thrown.getMessage shouldEqual "In Site{a + b => ...; a => ...}: Unavoidable nondeterminism: reaction {a(<4BE5...>) + b(2) => } is shadowed by {a(<4BE5...>) => }"
+
+    def expectsFromHash(hash: String) =
+      s"In Site{a + b => ...; a => ...}: Unavoidable nondeterminism: reaction {a(<$hash...>) + b(2) => } is " +
+      s"shadowed by {a(<$hash...>) => }"
+    val expected211 = expectsFromHash(hash211)
+    val expected212 = expectsFromHash(hash212)
+
+    Set(expected211, expected212) should contain oneElementOf List(thrown.getMessage)
```

Comment:
I'm puzzled as to why this didn't fail before. I already had a bunch of cases when 2.11 and 2.12 gave different hashes.